CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
 - Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
+- Adds possibility to filter by workflow status and search by workflow name to ``list`` command via ``--filter`` option.
```

Comment:
Aren't these better "Changes `list` command to add a possibility to filter..." etc?  I.e. not a new command, but changed behaviour via new options.